tensorflow/python/kernel_tests/quantization_ops/quantization_ops_test.py

```diff
@@ -381,6 +446,19 @@ def test_invalid_inputs(self):
       self.fail(
           "Did not raise an exception where it is expected to raise either "
           "a ValueError or errors.InvalidArgumentError.")
+class QuantizeDownAndShrinkRangeOpTest(test_util.TensorFlowTestCase):
+
+  @test_util.run_in_graph_and_eager_modes
+  def test_invalid_inputs(self):
+    inputs = constant_op.constant(
+        np.int32(0), shape=[3, 3, 3, 3], dtype=dtypes.qint32)
+
+    with self.assertRaisesRegex((ValueError, errors.InvalidArgumentError),
+                                "must be rank 0"):
+      self.evaluate(
+          math_ops.quantize_down_and_shrink_range(
+              input=inputs, input_min=[], input_max=4.0,
+              out_type=dtypes.quint8))
```

Comment:
```suggestion
```

Not in the branch, the CL changes a test that does not exist so we don't add a new test